### DISCUSSION TITLE
Unable to compile .zip via raylua_e

### DISCUSSION THREAD

**Discussion by SereysZ** [TSnake41/raylib-lua#6] [Q&A] [answered]
I am currently using "gcc -I"\LuaScripts\raylib-lua-master\src" raylua_e.c myFile.zip"
The output shows: "myFile.zip: error: unrecognized file format"
When i try to add the entire directory path as arguement it shows "unable to find file or directory" even though its there. Adding the extension .zip to the given path (myFile directory and .zip are in the same general path) results in the mentioned before error i.e. it is able to find the given file but for some reason not the entrie directory.
I dont know if i am missing any params or arguements to enter raylua_e in zip-mode or if i am doing something else entirely wrong but the documentation does not give any examples to raylua_e usage.

**Comment by TSnake41** [comment, marked as answer]
You are not supposed to use raylua_e source code to build your executable (and GCC has no clue on how raylua_e is designed and what to do with zip file).

You need to either download a [prebuild version of raylib-lua](https://github.com/TSnake41/raylib-lua/releases/download/v4.2a/raylua-4.2a-Win64-mingw.zip) and drag and drop your zip file onto the raylua_e executable, or by command-line
`raylua_e myFile.zip` (or `./raylua_e myFile.zip` depending on the operating system)

Or alternatively build raylib-lua using either classical `make` (or `mingw32-make`) or [saphire](https://github.com/TSnake41/saphire) and use the raylua_e in the same way as with prebuilt (e.g `./raylua_e myFile.zip`)
